tests/chart_tests/test_astronomer_config_syncer.py

```diff
@@ -0,0 +1,197 @@
+from tests.chart_tests.helm_template_generator import render_chart
+import pytest
+from tests import supported_k8s_versions
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+def test_astronomer_config_syncer_rbac_namespace_pools_disabled(kube_version):
```

Comment:
If you put all these test cases in a class you can parametrize the class, which will DRY up the test cases a bit

Reply:
Done, moved the other test files from this PR too